docs/_data/repoStatus.csv

```diff
@@ -1,74 +1,85 @@
-Element/Pattern,RHDS,WebRH,WebDMS,Adobe Target,
-12-column grid,,x,x,,
-Call to action - primary,x,x,x,,
-Call to action - primary video,x,,x,,
-Call to action - primary disabled,,,,,
-Call to action - secondary (ghost),,x,,,
-Call to action - secondary (ghost blue),,,,,
-Call to action - default,x,x,,,
-Call to action - default video,x,,x,,
-Call to action - brick,x,x,x,,
-Call to action - brick with icon,x,,,,
-Link with icon,,,,,
-Button - danger,x,,,,
-Button - primary,x,,,,
-Button - secondary,x,,,,
-Button - tertiary,x,,,,
-Button - link,x,,,,
-Button - play,x,,,,
-Button - close,x,,,,
-Skip to main content,,,,,
-Tabs - Open,x,,,,
-Tabs - Box,x,,,,
-Search bar,,,,,
-Search combo,,,,,
-Footer group,,,,,
-Accordion,,x,x,,
-Filter box,,x,,,
-Form - full,,x,,,
-Form - input group - money,,,,,
-Form - input group - email,,,,,
-Form - input group - username,,,,,
-Form - input group - date,,,,,
-Form - input group - search,,,,,
-Form - input group - help,,,,,
-Alert - inline,x,,,,
-Alert - inline variation,x,,,,
-Alert - toast,x,,,,
-Navigation,,x,,,
-Navigation Secondary,x,,,,
-Cookie banner,,,,,
-Logo bar,,,x,,
-Announcement bar,,,x,x,
-Dialog,x,,,,
-Video with thumbnail,,,x,,
-Quote,x,x,x,,
-Jump links - horizontal,,,,,
-Jump links - vertical,,,,,
-Fixed bottom banner,,,x,x,
-Audio player,,,x,,
-Tooltip,x,,,,
-Promo band - full,,x,,,
-Promo band - skinny,,x,,,
-Card - basic,,x,,,
-Card - lists,,x,,,
-Card - data/fast facts,,,x,,
-Card - logo,,x,,,
-Card - bar/with hat,,x,,,
-Card - icon,,x,,,
-Card - image,,x,,,
-Card - asset,,,x,,
-Card - quote,,x,,,
-Card - avatars,,,x,,
-Card - video,,,x,,
-Card - pricing,,,x,,
-Card - logo slider,,,x,,
-Card - name slider,,,x,,
-Avatar,x,,,,
-Tag - Red - filled and unfilled,x,,,,
-Tag - Orange - filled and unfilled,x,,,,
-Tag - Green - filled and unfilled,x,,,,
-Tag - Cyan - filled and unfilled,x,,,,
-Tag - Blue - filled and unfilled,x,,,,
-Tag - Purple - filled and unfilled,x,,,,
-Tag - Gray - filled and unfilled,x,,,,
+id,Element/Pattern,RHDS,WebRH,WebDMS,Adobe Target,
```

Comment:
I would alpha order this file by the id field now.  That way rows are easier to find and update

Reply:
Great idea! Thanks ✅